Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -73,7 +77,7 @@ - (void)identify:(SEGIdentifyPayload *)payload
     [self.amplitude setUserId:payload.userId];
     SEGLog(@"[Amplitude setUserId:%@]", payload.userId);
 
-    if ([self.traitsToIncrement count] > 0) {
+    if ([self.traitsToIncrement count] > 0 || [self.traitsToSetOnce count] > 0) {
```

Comment:
Maybe we should consider removing the else block (not in this PR but as a follow up) as this block will do the same thing if these settings are both empty.

Reply:
Hmm but I don't want to return within this block so that it checks `groups`. I can move groups up in another PR and then return out of this block and remove the else

Reply:
Hmmm, I think this is wrong. This going to call incrementOrSetTraits and setUserProperties. We don't want that. What I was I saying is I think we don't need to have this check, and don't need setUserProperties at all, the incrementOrSetTraits on it's own will handle correctly (no early return here).

Either way let's go back to what you had before and make the change in a *follow up PR*.